projects/messaging-semconv.md

```diff
@@ -0,0 +1,46 @@
+# Messaging Semantic Convention Stability
+
+## Description
+
+Get the messaging semantic conventions for metrics and traces to stability, and
+afterwards update existing messaging instrumentation across all OpenTelemetry
+repositories to conform with the stable conventions.
+
+## Deliverables
+
+* Mark the messaging semantic conventions as stable.
+* Update existing messaging instrumentation across all OpenTelemetry
+  repositories to conform with the stable conventions.
+
+## Staffing / Help Wanted
+
+Project lead
+
+* @pyohannes
```

Comment:
These warnings can be resolved by adding the names to the ignore list in https://github.com/open-telemetry/community/blob/main/.cspell.yaml.